tests/unit/useOnyxTest.ts

```diff
@@ -98,7 +98,7 @@ describe('useOnyx', () => {
         });
 
         it('should initially return null while loading non-cached key, and then return value and loaded state', async () => {
-            await StorageMock.setItem(ONYXKEYS.TEST_KEY, 'test');
+            await StorageMock.setItem(ONYXKEYS.TEST_KEY, {test: 'test'});
```

Comment:
Why these changes in this file?

Reply:
Typecheck is complaining in this file due to `OnyxValue` implementation:
```ts
type OnyxValue<TKey extends OnyxKey> = TKey extends CollectionKeyBase ? OnyxCollection<KeyValueMapping[TKey]> : OnyxEntry<KeyValueMapping[TKey]>;
```

second argument of `StorageMock.setItem` results in `OnyxCollection<unknown>`, that's why I changed it here. Any other ideas?

Reply:
I'm in favor of either trying to find a fix for `OnyxCollection<unknown>` case or using `// @ts-expect-error` because otherwise it would restrict our tests